src/components/screens/sprint-view-screen.tsx

```diff
@@ -29,30 +27,29 @@ const SprintViewScreen = () => {
   const [allocations, setAllocations] = useState<Allocations[]>([]);
   const [projects, setProjects] = useState<Projects[]>([]);
   const [timeEntries, setTimeEntries] = useState<number[]>([]);
-  const [loading, setLoading] = useState<boolean>(true);
-  const [myTasks, setMyTasks] = useState<boolean>(true);
-  const [filter, setFilter] = useState<string>("");
-  const sprintStartDate = getSprintStart((new Date()).toISOString());
+  const [loading, setLoading] = useState(false);
+  const [myTasks, setMyTasks] = useState(true);
+  const [filter, setFilter] = useState("");
+  const todaysDate = (new Date()).toISOString()
+  const sprintStartDate = getSprintStart(todaysDate);
   const sprintEndDate = getSprintEnd((new Date()).toISOString());
+  const columns = sprintViewProjectsColumns({allocations, timeEntries, projects});
+  const setError = useSetAtom(errorAtom);
 
   /**
-   * Get project data if user is logged in otherwise endless loading
+   * Get project data if user is logged in
    */
   useEffect(() => {
     setLoading(true);
```

Comment:
Its best to handle the loading states in one place/ function rather than splitting them between the use effect and the end of the function called in the use effect. In this case I would recommend placing both setLoadings within the function